### PR TITLE
add distance to likes and unseen matches

### PR DESCRIPTION
**Pull request by akhrail1996**
Continuing #662 

Distance is not added to seen matches because it allows continuous tracking of the same people (location + boolean for whether user logged into Since lately)

### CODE REVIEW

**Comment by akhrail1996** [comment]
GitHub actions seem to be down, but locally tests pass

image

File: lib/t/matches.ex [hunk 1]
```diff
@@ -42,21 +43,32 @@ defmodule T.Matches do
...
+    quote do
+      fragment(
+        "round(? / 1000)::int",
+        st_distance_in_meters(unquote(location1), unquote(location2))
+      )
+    end
+  end
 
-  @spec like_user(Ecto.UUID.t(), Ecto.UUID.t()) ::
+  defp default_location, do: %Geo.Point{coordinates: {0, 0}, srid: 4326}
```
Comment (ruslandoga) [inline lib/t/matches.ex:55]: ?

Comment (ruslandoga) [reply lib/t/matches.ex:55]: @akhrail1996 tests shouldn't affect the code they tests, please find another way

**Comment by akhrail1996** [comment]
@ruslandoga the approach to tests or the general approach (passing location to `like_user` and `list_matches`)?

**Pull request merged**